src/pam_hbac_obj.c

```diff
@@ -137,8 +144,33 @@ get_user_groups(const char *name, gid_t primary_gid,
         ret = 0;
         *ngroups_ptr = ngroups;
     }
-#else
-#error No known get-groups-for-user implementation found
+#else     // for systems lacking the above functions, tested on hpux only
```

Comment:
Created `ph_getgrouplist_fallback`. Think it's more suited since it's a replacement for the missing `getgrouplist` and `_getgroupsbymember` functions. But have no problem changing it.